### DISCUSSION TITLE
How to query cross all tenants?

### DISCUSSION THREAD

**Discussion by kehao95** [grafana/mimir#1635] [Help and support] [answered]
I'm new to Mimir, actually skipped Cortex. I just managed to run Mimir wishing to explore some internal metrics.
I hoped there is an admin role where I can explore all metrics from all tenants?
After exploring tickets, I still couldn't find it. I kept seeing the key word cross-tenants federation. But still don't know how to use it for querying.

**Comment by pstibrany** [comment, marked as answer]
Hello, and thank you for trying Grafana Mimir!

Query-federation works by explicitly specifying all queried tenants in `X-Scope-OrgID` header, separated by `|`.  There is currently no way to specify "query all tenants".

**Reply by kehao95** [reply]
Thank you for your explanation! I've created a PR https://github.com/grafana/mimir/pull/1689 to mention this in the doc.

**Reply by jriguera** [reply]
Is this supported in Opensource version?
Because I see [here](https://grafana.com/docs/mimir/latest/manage/secure/authentication-and-authorization/#:~:text=Grafana%20Mimir%20is%20a%20multi-tenant%20system%20where%20tenants,header%20of%20each%20request%2C%20for%20example%20X-Scope-OrgID%3A%20%3CTENANT-ID%3E.)  that `tenant-federation.enabled=true` should be set, but tenant-federation is an enterprise feature.
